pkg/master/master.go

```diff
@@ -89,6 +89,9 @@ type Config struct {
 	AdmissionControl       admission.Interface
 	MasterServiceNamespace string
 
+	// Map requests to contexts
+	RequestContextMapper api.RequestContextMapper
```

Comment:
Add a comment that this is exposed so that downstream projects can provide their own mappers